src/components/alarm_definitions.js

```diff
@@ -22,9 +22,9 @@ import MonascaClient from './monasca_client';
 export class AlarmDefinitionsPageCtrl {
 
   /** @ngInject */
-  constructor($scope, $injector, backendSrv, datasourceSrv, alertSrv) {
+  constructor($scope, $injector, backendSrv, datasourceSrv, alertSrv, monascaClientSrv) {
```

Comment:
Can we avoid modifying the code that we're testing?

Reply:
you can't test the grafana components without dependency injection in the constructor